01_temperature/temperature.rb

```diff
@@ -1 +1,12 @@
-#write your code here
+# this method takes in farenheit degrees to convert the temperature into celsius degrees
+  def ftoc(fahrenheit)
```

Comment:
This isn't a huge deal but method names should be easy to understand, I think the convention would be f_to_c or fahrenheit_to_celsius.